engine/order_manager.go

```diff
@@ -207,22 +204,23 @@ func (m *OrderManager) Cancel(ctx context.Context, cancel *order.Cancel) error {
 		err = fmt.Errorf("%v - Failed to cancel order: %w", cancel.Exchange, err)
 		return err
 	}
-	var od *order.Detail
-	od, err = m.orderStore.getByExchangeAndID(cancel.Exchange, cancel.ID)
+	// TODO: RACEY
```

Comment:
Please elaborate a little more on this TODO

Reply:
That was before I actually fixed it and I forgot to remove it.